modules/ossm-kiali-viewing-logs.adoc

```diff
@@ -0,0 +1,42 @@
+////
```

Comment:
The content for metrics is in the logs .adoc.  And the content for logs is in the metrics .adoc.